edx_sga/sga.py

```diff
@@ -146,6 +147,18 @@ class StaffGradedAssignmentXBlock(StudioEditableXBlockMixin, ShowAnswerXBlockMix
         help=_("When the annotated file was uploaded")
     )
 
+    @classmethod
+    def file_size_under_limit(cls, file_obj):
```

Comment:
It looks like we check this limit in the Javascript, and then again with the web server (it returns a 413 indicating the file is too large). I'm not sure we should add this check here.

Also this should be `file_size_over_limit`

Reply:
just in case someone use api to submit file, extra safty